HubSpot.NET.IntegrationTests/HubSpotAsyncIntegrationTestBase.cs

```diff
@@ -193,4 +195,28 @@ protected async Task<DealHubSpotModel> CreateTestDeal(string? dealName = null, d
         LineItemsToCleanup.Add(itemGetResponse.Id.Value);
         return (newLineItem, itemGetResponse);
     }
+
+    protected async Task<EventDefinition> GetOrCreateTestEventDefinition(string eventName = "test_event1",
+        string primaryObject = "CONTACT")
+    {
+        var existingEventDef = await CustomEventApi.GetByNameAsync<EventDefinition>(eventName);
```

Comment:
Other tests follow the approach of re-creating entity. But `EventDefinition` can't be re-created using same name. So selected the approach of getting existing event.

https://developers.hubspot.com/docs/guides/api/analytics-and-events/custom-events/custom-event-definitions

![image](https://github.com/user-attachments/assets/aa8e7557-aa18-40ae-a643-176c6c88a73b)